### DISCUSSION TITLE
Where to add vendor-specific names in Policy?

### DISCUSSION THREAD

**Discussion by schnuerle** [openmobilityfoundation/curb-data-specification#48] [Data Specification]
Should vendor specific names be included in user class or should it be incorporated somewhere else within the spec?

You could make one up for the user_classes, but that's not great or elegant. 

It could be added as a `vendor_name` field to the [Rule](https://github.com/openmobilityfoundation/curb-data-specification/tree/feature-release-work-1/curbs#rule) object so you can specify rules per vendor to make it free form text. But free form is flexible but not great for data analysis and consistency.

Maybe better that those options, it could instead be added as a list of `provider_ids` at the Policy level to show which provider a policy is applicable to. [This is what we do in MDS](https://github.com/openmobilityfoundation/mobility-data-specification/blob/main/policy/examples/README.md#provider-cap). If we go this route and use the naming in [this discussion](https://github.com/openmobilityfoundation/curb-data-specification/discussions/42#discussioncomment-1718010), we can use the field name `data_source_organization_ids` for consistency across CDS.

**Comment by jiffyclub** [comment]
+1 for taking the approach of MDS and having provider IDs on the policy. I'm confused about the suggested naming, though, I would expect policies to be applied to fleet operators, vehicle types, or business classes, not data vendors.

**Comment by bhamlinSDOT** [comment]
Agreed on MDS approach. data_source_organization_ids naming convention seems off though. I anticipate that in many instances vendor specific policies will be used by cities and vendors just for wayfinding purposes and not necessarily for the vendor to send data back or to be a data creator. I understand that the UUID's need to exist somewhere (and right now they would exist in data_source_organization_ids)  but there is a disconnect in these use cases which could be confusing?

**Comment by schnuerle** [comment]
Ok, provider_ids have been added as an option at the Policy level. 

https://github.com/openmobilityfoundation/curb-data-specification/blob/feature-release-work-1/curbs/README.md#policy

Going with this name for now, but may be updated based on other discussions.
